Sources/UnidirectionalBinding.swift

```diff
@@ -10,17 +10,31 @@ precedencegroup BindingPrecedence {
 
 infix operator <~ : BindingPrecedence
 
+#if swift(>=5.7)
+/// Describes a source which can be bound.
+public protocol BindingSource<Value>: SignalProducerConvertible where Error == Never {}
+#else
 /// Describes a source which can be bound.
 public protocol BindingSource: SignalProducerConvertible where Error == Never {}
+#endif
 extension Signal: BindingSource where Error == Never {}
 extension SignalProducer: BindingSource where Error == Never {}
 
+#if swift(>=5.7)
+/// Describes an entity which be bond towards.
```

Comment:
```suggestion
/// Describes an entity which can be bound towards.
```

This typo is also in the original below

Reply:
Done. Fixed it in a couple other spots that seemed incorrect as well